src/error.ts

```diff
@@ -453,7 +468,22 @@ Object.defineProperty(Error.prototype, 'toJSON', {
     if (this.bundle != null) json.bundle = this.bundle;
     if (Array.isArray(this.suberrors)) json.suberrors = this.suberrors.map((item: Error) => item.toJSON());
```

Comment:
I wonder if this is even necessary, cause I would expect the JSON.stringify() to actually go through all of the items in the array and call toJSON on them. I wonder why we are doing it manually here, but @skvelymake taught me not to touch such kind of things here :D